mix_helpers.exs

```diff
@@ -252,13 +252,28 @@ defmodule Mix.Appsignal.Helper do
   end
 
   defp download_options do
-    options = [
-      ssl_options:
-        [
-          verify: :verify_peer,
-          cacertfile: priv_path("cacert.pem")
-        ] ++ tls_options() ++ customize_hostname_check_or_verify_fun()
-    ]
+    default_cacert_file_path = priv_path("popo.pem")
+
+    options =
+      case check_cacert_access(default_cacert_file_path) do
+        :ok ->
+          [
+            ssl_options:
+              [
+                verify: :verify_peer,
+                cacertfile: default_cacert_file_path
+              ] ++ tls_options() ++ customize_hostname_check_or_verify_fun()
+          ]
+
+        {:error, message} ->
+          Logger.warn(
+            "The cacert file path: #{default_cacert_file_path} is not accessible. " <>
+              "Reason: #{inspect(message)}. " <>
+              "Using system defaults instead."
+          )
+
+          []
```

Comment:
Nitpick, feel free to ignore: This does work fine and loads the default SSL config, but since [the SSL implementation in Hackney is a bit finicky](https://nts.strzibny.name/hackney-ssl-configuration/), I worry about passing two differently shaped SSL config objects to it and accidentally introducing different behaviours. It should be possible to pass `:certifi.cacertfile()` to use the default CA file.